tests/appsec/rasp/test_stack_traces.py

```diff
@@ -0,0 +1,81 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import features, weblog, interfaces, scenarios, rfc, missing_feature
+
+
+def validate_stack_traces(request):
+    spans = [s for _, s in interfaces.library.get_root_spans(request=request)]
+    assert spans, "No spans to validate"
+
+    for span in spans:
+        assert "_dd.appsec.json" in span["meta"], "'_dd.appsec.json' not found in 'meta'"
+
+        json = span["meta"]["_dd.appsec.json"]
+        assert "triggers" in json, "'triggers' not found in '_dd.appsec.json'"
+
+        triggers = json["triggers"]
+
+        # Find all stack IDs
+        stack_ids = []
+        for event in triggers:
+            if "stack_id" in event:
+                stack_ids.append(event["stack_id"])
+
+        # The absence of stack IDs can be considered a bug
+        assert len(stack_ids) > 0, "no 'stack_id's present in 'triggers'"
+
+        assert "meta_struct" in span, "'meta_struct' not found in span"
+        assert "_dd.stack" in span["meta_struct"], "'_dd.stack' not found in 'meta_struct'"
+        assert "exploit" in span["meta_struct"]["_dd.stack"], "'exploit' not found in '_dd.stack'"
+
+        stack_traces = span["meta_struct"]["_dd.stack"]["exploit"]
```

Comment:
Same here. If you expect at least one...

```suggestion
        stack_traces = span["meta_struct"]["_dd.stack"]["exploit"]
        assert stack_traces
```